content/en/docs/code/cri-api-dev-policies.md

```diff
@@ -0,0 +1,126 @@
+---
+content_type: "reference"
+title: Kubernetes feature development and container runtimes
+weight: 55
+---
+
+CRI is a plugin interface which enables kubelet to use a wide variety of container runtimes,
```

Comment:
nit: do we have some doc guidelines whether we should use "enables kubelet" or "enables the kubelet"?

I prefer the former myself, but the latter seems to be used so often that I thought it was a convention that I just hadn't trained myself to follow...

Reply:
fixed here and in other document